app/views/users/new.html.erb

```diff
@@ -0,0 +1,49 @@
+<%
+=begin%>
+<h1>User Registration</h1>
+<form action='/users' method='POST'> <br>
+  <label for='username'>Username</label> <br>
+  <input type='text' id='username' name='user[username]'> <br>
+  <label for='email'>Email</label> <br>
+  <input type='text' id='email' name='user[email]'> <br>
+  <label for='password'>Password</label> <br>
+  <input type='password' id='password' name='user[password]'> <br>
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <input type='submit'>
+</form>
+<%
+=end%>
+
+<%
+=begin%>
+<%= form_tag("/users", method: "post") do%>
```

Comment:
`method: "post"` is default value for this already so you don't need to define it again. It's only often needed for other actions like `DELETE`. It should be => `<%= form_tag("/users") do%>`